src/main/java/com/cleanroommc/groovyscript/compat/mods/advancedrocketry/BaseMultiblockRegistry.java

```diff
@@ -0,0 +1,48 @@
+package com.cleanroommc.groovyscript.compat.mods.advancedrocketry;
+
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.SimpleObjectStream;
+import net.minecraft.item.ItemStack;
+import zmaster587.libVulpes.interfaces.IRecipe;
+
+import java.util.*;
+
+public abstract class BaseMultiblockRegistry extends BaseRegistry {

```

Comment:
uhm. now that i think about it a bit, is there any need for `BaseMultiblockRegistry` to exist if it only serves as something to contain the `BaseRegistry.RecipeBuilder` class?

Reply:
That's a very good note. I'll adjust this later today.

Reply:
done, I think. didn't test it.